addons/project/models/project_project.py

```diff
@@ -365,12 +365,13 @@ def _compute_access_instruction_message(self):
                 project.access_instruction_message = ''
 
     @api.model
-    def _map_tasks_default_values(self, task, project):
-        """ get the default value for the copied task on project duplication """
+    def _map_tasks_default_values(self, project):
+        """ get the default value for the copied task on project duplication.
+        The stage_id, name field will be set for each task in the overwritten copy_data function in project.task """
         return {
-            'stage_id': task.stage_id.id,
-            'name': task.name,
-            'state': task.state,
+            'stage_id': False,
+            'name': False,
```

Comment:
Why do we need to set those fields to False? 🤔

Reply:
dunno, it was already present in the original version. I did not test if made any difference with or without, tbh. I'll try removing it and see if it breaks things :man_shrugging:

Reply:
edit : after some testing, it looks like it was indeed unneeded. I removed it, and the runbot is green.